services/core-web/src/components/mine/ExplosivesPermit/MineExplosivesPermitTable.tsx

```diff
@@ -103,16 +82,8 @@ const MineExplosivesPermitTable: FC<RouteComponentProps & MineExplosivesPermitTa
   ...props
 }) => {
   const isAdmin = useSelector((state) => userHasRole(state, Permission.ADMIN));
-  const editIcon = isFeatureEnabled(Feature.ESUP_PERMIT_AMENDMENT) ? (
-    <EditOutlined />
-  ) : (
-    <VioletEditIcon className="padding-sm" />
-  );
-  const viewIcon = isFeatureEnabled(Feature.ESUP_PERMIT_AMENDMENT) ? (
-    <EyeOutlined />
-  ) : (
-    <EyeOutlined className="padding-sm icon-lg icon-svg-filter" />
-  );
+  const editIcon = <EditOutlined />;
```

Comment:
Could probably put these inline where they're used now.

Reply:
I considered it, but there's 4 instances, so I decided I prefer the variable.

Reply:
Okay, works for me.